### DISCUSSION TITLE
Stuttering playback on Firestick using SlingerPlayer

### DISCUSSION THREAD

**Discussion by cgcmgr** [GerryDazoo/Slinger#588] [General]
Hey everyone,

I have slingbox_server 4.01 running on a PC at home and I'm running SlingerPlayer on a Firestick in a remote location.  My upload speed at home is registering 940.66 Mbps (FIOS Gigabit) and the download speed at the remote location is registering at 100.34 Mbps, when hard wired.  This is my first time testing the set up remotely and it seems to work perfectly when watching normal TV.  However, I was watching a hockey game over the weekend and although watchable, there was a lot of stuttering.  It wasn't fluid playback.  The Firestick is connected to my 5Ghz Wi-Fi network and I'm not sure of the download speed the Firestick is getting on the Wi-Fi as I didn't run a speed test, don't ask me why.  The Firestick is actually connecting to my home VPN so all traffic is running through the VPN from my house to the remote location.

I have a desktop PC in this remote location as well and when running SlingerPlayer on it, the hockey game was fluid.  This PC is hardwired.  The desktop is in a different room than the main TV so I can't connect it to the TV.

So, I'm wondering what may be causing the stuttering on the Firestick.  Is it the fact that it is on Wi-Fi or do you think changing a setting or two in my Config file may help?  My Slingbox is a M1 and I have the Config file as such:
Resolution=16 which should equate to 1920x1080
Framerate=30
VideoBandwidth=4000
VideoSmoothness=63
AudioBitRate=48

Should I reduce the resolution to 1280x720 or maybe the Video Bandwidth?  Will increasing the Video Smoothness help?

I have a Dell Optiplex 7070 micro desktop that I have configured here at home and I will be using that in the remote location going forward in place of the Firestick and this PC will be hardwired.  I'd still like to use the Firestick elsewhere and would like sports to play smoothly.

Thanks for any and all suggestions,
Chris

**Comment by kdurkee-cohere** [comment]
When you have stuttering you can usually skip back on the player 5-10 seconds behind the stream and it usually will play smooth after that. 

I found PlayerXtreme for iOS and Android to give the best playback experience and for Laptop or Desktop Win/Mac StellarPlayer works very well

**Reply by cgcmgr** [reply]
Appreciate the reply.  I will try your suggestions.

**Reply by as400jockey** [reply]
how do / can you skip back on SlingerPlayer on Firestick?

**Reply by GerryDazoo** [reply]
You can't.
